workspaces/global-header/plugins/global-header/src/components/MenuItemLink/MenuItemLink.tsx

```diff
@@ -46,8 +48,14 @@ export const MenuItemLink = ({
         textDecoration: 'none',
         width: '100%',
       }}
+      {...(isExternalLink ? { target: '_blank' } : {})}
```

Comment:
I know this is old code, but can we change the import for Link from `react-router-dom` to `@backstage/core-components`?

It automatically adds target blank (and analytics!) for external links. Then we could remove this target: _blank prop here.

Reply:
Done.